AsyncDisplayKitTests/ASStackLayoutSpecSnapshotTests.mm

```diff
@@ -41,8 +42,8 @@ - (void)setUp
                         ];
   for (ASStaticSizeDisplayNode *subnode in subnodes) {
     subnode.staticSize = subnodeSize;
-    subnode.flexGrow = flex;
-    subnode.flexShrink = flex;
+    subnode.layoutOptions.flexGrow = flex;
```

Comment:
Can you delete every addition of .layoutOptions in this file?  Aren't these supposed to be forwarded automatically?